documentation/book/assembly-zookeeper-connection.adoc

```diff
@@ -0,0 +1,13 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='assembly-zookeeper-connection-{context}']
+
+= Zookeeper connection
+
+Zookeeper services are secured with encryption and authentication and are not intended to be used by external applications that are not part of AMQ Streams.
+
+However, if you want to use Kafka CLI tools that require a connection to Zookeeper, you can use a terminal inside a Kafka container and connect using `localhost:2181` as the Zookeeper address.
```

Comment:
An example of a Kafka CLI tool, such as the kafka-topics tool, might be helpful here.

Reply:
There is an example in the procedure right below this. But I guess it might not hurt to have it here as well.